### PR TITLE
Project Blogs

### PR DESCRIPTION
**Pull request by jfucci**
Added blogs for individual projects. Posts can be added but not edited or deleted at this point. The revert is only due to my starting on a way to add users to a project before realizing that it was different from how the ui is now implemented. Also, addresses #30 .

### CODE REVIEW

**Comment by agundy** [comment]
Can you fix it so that is passes Travis CI? Thanks!

File: server/api/post/post.model.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
+
+var PostSchema = new Schema({
+  title: String,
+  content: String,
+  project: String,
```
Comment (seveibar) [inline server/api/post/post.model.js]: What if the project name changes? Should this be an id?

**Comment by seveibar** [comment]
Awesome PR (at least w/ code stylistically). I'll pull it in and check functionality later (unless someone else gets to it haha)

File: server/api/post/post.model.js (hunk 1 above)
Comment (jfucci) [inline server/api/post/post.model.js]: Good point. Changed to use ids.

**Comment by seveibar** [comment]
I'm not sure this is ready for merge- the links on the projects page don't seem to go to a blog.

**Pull request merged**